haas/rest.py

```diff
@@ -164,10 +170,13 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        methods = method if isinstance(method, list) else [method]
```

Comment:
Suggestion:

1. rename the parameter method to methods
2. Just have:

```python
if not isinstance(methods, list):
    methods = [methods]
```

...I find python's if expressions really hard to read; they look inside-out to me.

One alternative is to just get rid of the if entirely, and go change all of the calls to this; more annoying to do in the short term, but it makes our own glue code smaller/simpler, and my preference is to push as much of the misc. web framework goop down into flask as we can. I don't feel strongly about doing this now though.